### DISCUSSION TITLE
TextInput onContentSizeChange behaves different

### DISCUSSION THREAD

**Discussion by mirik999** [reactwg/react-native-new-architecture#245] [Q&A]
Hi everyone, after upgrading my expo app to the new architecture and RN 0.76.1, text input don't call onContentSizeChange when it gets to the second line. It only fires once on mount. I could not find any info about it, please explain how to fix it.

**Comment by cortinico** [comment]
Please open an issue on https://github.com/facebook/react-native/issues and we can look into it.
